data-raw/create_bigquery_from_postgres.R

```diff
@@ -3,16 +3,22 @@ library(data.table)
 library(glue)
 library(bigrquery)
 
-createBigQueryFromPostgres = function(pgDbName = 'pmdb', project = 'pmparser-test', dataset = 'pmdb', tableName, chunkSize = 15000L, overwriteTable = FALSE, tables = c()){
+#tables variable is a named list where the name is the table name and the value is the chunk size you want to process. Chunk sizes of 0 or NA will be removed from list
```

Comment:
The `tables` variable seems redundant with `chunkSize` and its name is misleading. Figure out how to use only the `chunkSize` argument.